README.md

```diff
@@ -7,152 +7,31 @@ This is a web application that:
  2. Fetches and displays build history of the ports.
  3. Collects and displays installation statistics (system configurations and installed ports) from users who opt-in by
     installing the port `mpstats`.
+ 4. Allows to follow ports and receive notifications when a port gets updated.
     
 It aims at being an all-one-place for users to find relevant ports and for maintainers to monitor their ports and
 understand their user base. The app is running in production at [ports.macports.org](https://ports.macports.org).
  
-More info at [docs/1-INFORMATION.md](docs/1-INTRODUCTION.md)
+More info at [docs/INFORMATION.md](docs/INTRODUCTION.md)
 
 ### Architecture
 
-The web application is based on the [Django Framework](http://djangoproject.com) utilising a PostgreSQL database.
-
-The app is supposed to be deployed in a docker container with nginx and uWSGI serving the content.
+The web application is based on:
+ - [Django Framework](https://www.djangoproject.com)
+ - [PostgreSQL Database](https://www.postgresql.org)
+ - [Solr](https://lucene.apache.org/solr/)
+ - [Memcached](http://memcached.org)
  
-This web application has been implemented during the [Google Summer of Code 2019](https://summerofcode.withgoogle.com) 
-mentored by the [MacPorts](https://www.macports.org) organisation.
-
-**Student**: [Arjun Salyan](https://github.com/arjunsalyan) <br>
```

Comment:
So you removed your name and kept `HISTORY.md` empty?

Reply:
I was simplifying the readme with all the information in separate links. But never really got to the point of finishing all the files. I hope to be able to do it soon.

Reply:
I'm aware that the docs were not finished, and I don't expect them to be complete. It's just strange that you completely remove your name ;)